### DISCUSSION TITLE
Request for Single Sign-On feature for Facebook, Apple, and Google on Shopify Fluxstore app

### DISCUSSION THREAD

**Discussion by abdullahmujahidali** [inspireui/support#5736] [New Feature/Ideas Requests] [answered]
I would like to request the addition of a Single Sign-On (SSO) feature for Facebook, Apple, and Google on the Shopify Fluxstore app. As a user, I find it tedious to enter my login details every time I access the app, and I believe that many other users feel the same way.

SSO would streamline the login process by allowing users to log in with their Facebook, Apple, or Google accounts. This would not only save time but also increase the security of the login process, as users would not need to remember multiple login details.

I understand that this feature is already available on platforms like WordPress, and I believe that adding it to the Shopify Fluxstore app would make it more competitive and user-friendly.

Thank you for considering my request. I look forward to seeing this feature implemented in the future.

**Comment by vadainnovation** [comment]
Yes this feature should be on priority

**Reply by abdullahmujahidali** [reply, marked as answer]
Feature is in development
